### PR TITLE
minor formatting of routes

### PR DESCRIPTION
**Pull request by 2is10**
@yipingliao

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/route.js [hunk 1]
```diff
@@ -63,68 +63,66 @@ angular.module('kifi')
...
         templateUrl: 'search/search.tpl.html',
         controller: 'SearchCtrl',
-
-        // SearchCtrl manually reloads search results on search input change in a debounced function.
-        reloadOnSearch: false
+        reloadOnSearch: false  // controller handles search query changes itself
       })
       .state('userProfile', {
-        url: '/:username?upb',
+        url: '/:username',
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/route.js:69]: `upb` does not seem to be referenced anywhere else

File: kifi-backend/modules/shoebox/angular/src/route.js [hunk 2]
```diff
@@ -63,68 +63,66 @@ angular.module('kifi')
...
         templateUrl: 'userProfile/userProfile.tpl.html',
         controller: 'UserProfileCtrl',
         'abstract': true
       })
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/route.js:73]: The edits below are just flattening the indentation. Didn’t want to mix subsequent edits in with this simple formatting change.

File: kifi-backend/modules/shoebox/angular/src/route.js (hunk 1 above)
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/route.js:69]: Oops, thanks. That should have been removed as part of removing the hooks on profiles_beta experiment.

**Comment by yipingliao** [comment]
:+1:

**Pull request merged**